### DISCUSSION TITLE
SMART on FHIR repo?

### DISCUSSION THREAD

**Discussion by anton-kurilov** [keycloak/keycloak#26753] [Q&A]
Hello keycloak folks, in the HL7 FHIR community there is an extension project that has fallen to the wayside: https://github.com/Alvearie/keycloak-extensions-for-fhir 

But that still has some interest from the community: https://chat.fhir.org/#narrow/stream/179170-smart/topic/Keycloak.20for.20SMART.20authz  

We are currently looking for new possible repo location to restart this project for the community and was wondering if this is something that should belong under keycloak quickstarts or something? And how we can kick off the discussion/process to push there if possible.

Thanks!

**Comment by XcrigX** [comment]
To add some context for those unfamiliar, SMARTonFHIR is a set of patterns/extensions on OAuth/OIDC used to enable secure access to electronic health records (EHRs) via API. 
Many countries now mandate that EHR systems support SMART. This helps enable individuals' access to their own data, as well enables transmitting data between systems involved in a patient's care. 

The linked github project above has a good readme explanation of the things required by SMART which are not supported out of the box by Keycloak (or really any existing IAM solutions). 

You can find more on the specification here : 
https://hl7.org/fhir/smart-app-launch/index.html
